src/ofxValuePlotter.h

```diff
@@ -27,15 +33,15 @@ class ofxValuePlotter: public ofxBaseGui {
     bool setValue(float mx, float my, bool bCheckBounds){return false;}
     void render();
     void generateDraw();
+    void update(float & value);
     ofPath bg;
     ofVboMesh textMesh;
-    vector<float> buffer;
+    std::vector<float> buffer;
     int plotSize;
     ofPath plot;
-    float lastVal;
     float minVal, maxVal;
     bool autoscale;
-    int decimalPlace=3;
-    ofParameter<string> label;
+    int decimalPlace;
+    ofParameter<float> value;
 
```

Comment:
you are right, this parameter and label mixup should be changed in my version as well